polls/views.py

```diff
@@ -1,15 +1,18 @@
+from django.contrib.auth import get_user_model
 from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
 from django.shortcuts import redirect, render
+from django.urls import reverse
 from django.utils import timezone
 from django.views import View
 from django.views.generic import TemplateView
-from rest_framework import permissions, status
+from rest_framework import generics, mixins, permissions, status
 from rest_framework.authentication import TokenAuthentication
```

Comment:
TokenAuthentication is imported and unused, it should be commented out or better still removed from the codebase.